database/src/test/scala/za/co/absa/atum/database/runs/GetPartitioningMeasuresTest.scala

```diff
@@ -103,7 +103,7 @@ class GetPartitioningMeasuresTest extends DBTestSuite {
       .execute { queryResult =>
         val results = queryResult.next()
         assert(results.getInt("status").contains(41))
-        assert(results.getString("status_text").contains("Partitioning not found"))
+        assert(results.getString("status_text").contains("The partitioning does not exist."))
```

Comment:
this is not what we are using all over the place - it's what was originally there i.e. `Partitioning not found`